### PR TITLE
Update web interface to play OGA audio plus update the image formats supported by it.

### PR DESCRIPTION
**Pull request by valib**
Tested on Firefox and Chrome. This audio format is not supported by MSIE and EDGE.

### CODE REVIEW

**Comment by SubJunk** [comment]
@valib the formatting change looks good, reducing the duplication of MIME type definitions.
We already detect which browser is connected, maybe this change could be conditional upon the browser being Firefox or Chrome, to prevent passing a broken video to other browsers?

**Comment by valib** [comment]
@SubJunk this could be done only when we introduce the browser's.conf and rewrite the `WebRendrer` where is set configuration for all browsers in the https://github.com/UniversalMediaServer/UniversalMediaServer/blob/11acac7c3bd521219f41d97a60ef50c0f0884722/src/main/java/net/pms/configuration/WebRender.java#L115
because the configuration setting is done during initialization and there is not possible to distinguish the current browser and set it properly.

File: src/main/java/net/pms/remote/RemoteUtil.java [hunk 1]
```diff
@@ -179,12 +173,15 @@ public static boolean directmime(String mime) {
...
-				mime.equals(MIME_JPG)
+				mime.equals(HTTPResource.MP4_TYPEMIME) ||
+				mime.equals(HTTPResource.WEBM_TYPEMIME) ||
+				mime.equals(HTTPResource.OGG_TYPEMIME) ||
+				mime.equals(HTTPResource.AUDIO_OGA_TYPEMIME) ||
+				mime.equals(HTTPResource.AUDIO_MP3_TYPEMIME) ||
+				mime.equals(HTTPResource.PNG_TYPEMIME) ||
+				mime.equals(HTTPResource.JPEG_TYPEMIME) ||
+				mime.equals(HTTPResource.GIF_TYPEMIME) ||
+				mime.equals(HTTPResource.TIFF_TYPEMIME)
```
Comment (Sami32) [inline src/main/java/net/pms/remote/RemoteUtil.java]: Except for Safari users, all the others should get TIFF issue.

**Comment by valib** [comment]
@SubJunk I tested it on Firefox and Chrome and it mostly works. The TIFF format is strange and there is not a web browser which supports all of combinations of this this format. I think that this the best way how to offer users to show the TIFF pictures.

**Comment by Sami32** [comment]
https://github.com/openseadragon/openseadragon/issues/1380
https://en.wikipedia.org/wiki/Comparison_of_web_browsers#Image_format_support
https://bugzilla.mozilla.org/show_bug.cgi?id=160261

**Comment by Nadahar** [comment]
TIFF are already handled by the automatic image conversion so I can view them without this PR. I don't think saving the small conversion time is worth losing support for many of them.

**Pull request merged**